### PR TITLE
Frontend refactoring

### LINKED ISSUE #238: Refactor CSS and HTML
I will open up a branch cleaning old unnecassary CSS classes. I will also clean up workaround and etc.

### PR DESCRIPTION
**Pull request by JanSchneider1**
The code isn't perfect after this. But I changed the formatting, gave the
Coding a constistent style and removed unneccessary code.

Closes #238

### CODE REVIEW

**Comment by ccaylak** [comment]
Why do some files begin with an underscore?

File: src/main/resources/static/css/matchresult.less [hunk 1]
```diff
@@ -1,5 +1,7 @@
-@import "_colorscheme.less";
-@import "searchbar.less";
+@import "_colorscheme";
+@import "_mediaqueries";
+@import "_background";
+@import "_searchbar";
```
Comment (Flo4l) [inline src/main/resources/static/css/matchresult.less]: I prefer that the filenames are written out - including the filetype.
We already have that convention for exampple in the controllers where the `.html` ending specifys the type.
So does everyody know what kind of file this is.

Please append them to the other files too, i won't comment everyone of them.

Comment (Flo4l) [reply src/main/resources/static/css/matchresult.less]: Everything else looks fine to me

Comment (JanSchneider1) [reply src/main/resources/static/css/matchresult.less]: > 
> 
> Why do some files begin with an underscore? And wouldn't this PR close #238?

Files containing meta information meaning that there are not used by a 'physical' site begin with an underscore to show that. Less will compile them an insert the rules via the @import

**Pull request merged**